docs/sphinx/source/api/Specification.rst

```diff
@@ -201,3 +201,4 @@ Allowed field      Data Type             Description
 **receiver_port**  str                   The id of the InputPort on the receiver `Node <#node>`__
 **parameters**     Union[Any, NoneType]  Dictionary of parameters for the edge.
 =================  ====================  ============================================================================================================
+
```

Comment:
This change and the one above are probably not required and should be removed.

Run `pre-commit run --all-files` again on your local checkout of this branch and these should be reverted, and then commit again.

Reply:
Please disregard this pull request. There were alot of problems with this branch hence i created a new branch